src/timer/windows_timer.rs

```diff
@@ -48,24 +52,24 @@ pub struct CPUTimer {
 
 impl CPUTimer {
     pub unsafe fn start_suspended_process(child: &process::Child) -> Self {
-        let child_handle = child.as_raw_handle().cast();
+        let child_handle = child.as_raw_handle() as HANDLE;
 
         // SAFETY: Creating a new job object is safe
         let job_object = unsafe { CreateJobObjectW(ptr::null_mut(), ptr::null_mut()) };
-        assert!(!job_object.is_null(), "CreateJobObjectW failed");
+        assert!(job_object != 0, "CreateJobObjectW failed");
 
         // SAFETY: The job object handle is valid
         let ret = unsafe { AssignProcessToJobObject(job_object, child_handle) };
         assert!(ret != 0, "AssignProcessToJobObject failed");
 
-        #[cfg(windows_process_extensions_main_thread_handle)]
+        #[cfg(feature = "windows_process_extensions_main_thread_handle")]
```

Comment:
If I understand/recall correctly, the intention was that this version could be used for nightly-compiler builds (with `windows_process_extensions_main_thread_handle` additionally activated), and once that feature is stabilized, we can remove the alternative `not(feature = …)` version below and make this the main code path.

Reply:
Yes, this is the plan 👍